### PR TITLE
Add BreadrumbOnRails Gem

### PR DESCRIPTION
**Pull request by jessedoyle**
:elephant:
* Write our own custom builder class for the BreadcrumbsOnRails gem
  to properly create our Foundation breadcrumbs.
* Write unit tests for our builder implementation.
* Include BreadcrumsOnRails v2.3.0 as a dependency.

### CODE REVIEW

File: lib/ama_layout/breadcrumb_builder.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  class BreadcrumbBuilder < BreadcrumbsOnRails::Breadcrumbs::Builder
+    def render
+      @elements.map { |e| render_element(e) }.join(@options[:separator])
+    end
+
+  private
+
+    def render_element(element)
+      name = compute_name(element)
+      path = element.path && compute_path(element) || '#'
```
Comment (MathieuGilbert) [inline lib/ama_layout/breadcrumb_builder.rb]: Can you use the `name` variable here?

File: spec/ama_layout/breadcrumb_builder_spec.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
+describe AmaLayout::BreadcrumbBuilder do
+  let(:view_context) { ActionView::Base.new }
+
+  describe '#render' do
+    let(:builder) { AmaLayout::BreadcrumbBuilder.new(view_context, crumbs) }
+
+    context 'no separator specified' do
+      let(:crumbs) do
```
Comment (MathieuGilbert) [inline spec/ama_layout/breadcrumb_builder_spec.rb]: What do you think about exposing the crumbs separately at the top, then you'd only have to override one of them in the disabled test?

```
  let(:foo_crumb) { BreadcrumbsOnRails::Breadcrumbs::Element.new('Foo', '/foo') }
  let(:bar_crumb) { BreadcrumbsOnRails::Breadcrumbs::Element.new('Bar', '/foo/bar') }
  let(:crumbs) { [foo_crumb, bar_crumb] }
```

**Comment by MathieuGilbert** [comment]
💩

File: lib/ama_layout/breadcrumb_builder.rb (hunk 1 above)
Comment (jessedoyle) [inline lib/ama_layout/breadcrumb_builder.rb]: I used the code [here](https://github.com/weppos/breadcrumbs_on_rails/blob/master/lib/breadcrumbs_on_rails/breadcrumbs.rb#L79-L100) as an example.

**Comment by jessedoyle** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**